Merge entities onto main project

Guztaver opened the pull request:
this is necessary to use search the musics, and work properly on the requested material

Guztaver committed:
Add initial project files including Playlist and Song classes, XML configurations, and repository settings

Guztaver commented:
Comentei em inglês por força de hábito, mas, vocês acham que tá bom? Ou na pasta de entidades fica esquisito pra esse projeto?

matheusmontev reviewed (dismissed):
Gostei não achei nada de errado

GustavoADiniz reviewed (dismissed):
Esta satisfatorio.

Guztaver committed:
feat: edit the name of components to show all

Guztaver commented:
Tive que adicionar o nome do pessoal no README.md, acabei me esquecendo desse pequeno detalhe!

GustavoADiniz reviewed (commented):
Verifiquei que tanto meu nome quanto o seu e de matheus estão por completo achoi que seria interresante adicionar os sobrenomes de Alex e de Yves

Guztaver committed:
feat: Implement music player with repository functionality

Add song repository management for storing and organizing audio files.
Develop complete player functionality including playlist creation,
editing and playback with controls (pause, next, previous).
Update dependencies and team contact information in README.

Guztaver commented:
@GustavoADiniz melhor agora?

GustavoADiniz reviewed (approved):
Sim. Bom trabalho.

SAGA3568 reviewed (commented):
O codigo está deverás fabuloso.

SAGA3568 reviewed (approved):
Excelente!
